### PR TITLE
Extensionless dicom files upload

### PR DESCRIPTION
**Pull request by pritamrungta**
- Extensionless dicom files upload
- Update docs for enums
- Restrict project creation to use Taxonomy V2

### CODE REVIEW

File: redbrick/common/enums.py [hunk 1]
```diff
@@ -14,7 +14,16 @@ class StorageMethod:
 
 
 class TaskStates(str, Enum):
-    """Potential states of task status."""
```
Comment (ben-stewart-rb) [inline redbrick/common/enums.py:17]: Potential Task Status States

File: redbrick/common/enums.py [hunk 2]
```diff
@@ -14,7 +14,16 @@ class StorageMethod:
...
+    """Potential states of task status.
+
+    - ``UNASSIGNED`` - The task has not been assigned to any worker.
```
Comment (ben-stewart-rb) [inline redbrick/common/enums.py:19]: The Task has not been assigned to a Project Admin or Member

File: redbrick/common/enums.py [hunk 3]
```diff
@@ -14,7 +14,16 @@ class StorageMethod:
...
+    - ``ASSIGNED`` - The task has been assigned to a worker but not yet started.
```
Comment (ben-stewart-rb) [inline redbrick/common/enums.py:20]: The Task has been assigned to a Project Admin or Member but work has not begun on it

File: redbrick/common/enums.py [hunk 4]
```diff
@@ -14,7 +14,16 @@ class StorageMethod:
...
+    - ``IN_PROGRESS`` - The task is currently being worked on by a worker.
```
Comment (ben-stewart-rb) [inline redbrick/common/enums.py:21]: The Task is currently being worked on by a Project Admin or Member

File: redbrick/common/enums.py [hunk 5]
```diff
@@ -14,7 +14,16 @@ class StorageMethod:
...
+    - ``COMPLETED`` - The task has been completed successfully.
```
Comment (ben-stewart-rb) [inline redbrick/common/enums.py:22]: Capitalize Task

File: redbrick/common/enums.py [hunk 6]
```diff
@@ -14,7 +14,16 @@ class StorageMethod:
...
+    - ``PROBLEM`` - The task has encountered an issue and cannot be worked upon.
```
Comment (ben-stewart-rb) [inline redbrick/common/enums.py:23]: Assuming this is the "Raise Issue" state...

A Project Admin or Member has raised an Issue regarding the Task, and work cannot continue until the Issue is resolved by a Project Admin

File: redbrick/common/enums.py (hunk 3 above)
Comment (pritamrungta) [reply redbrick/common/enums.py:20]: @ben-stewart-rb  can this one be done in less than 80 characters?

File: redbrick/common/enums.py [hunk 7]
```diff
@@ -14,7 +14,16 @@ class StorageMethod:
...
+    - ``SKIPPED`` - The task has been skipped.
```
Comment (ben-stewart-rb) [inline redbrick/common/enums.py:24]: The Task has been skipped

File: redbrick/common/enums.py [hunk 8]
```diff
@@ -14,7 +14,16 @@ class StorageMethod:
...
+    - ``STAGED`` - The task has been saved as draft.
```
Comment (ben-stewart-rb) [inline redbrick/common/enums.py:25]: The Task has been saved as a Draft.

File: redbrick/common/enums.py [hunk 9]
```diff
@@ -26,7 +35,12 @@ class TaskStates(str, Enum):
 
 
 class ReviewStates(str, Enum):
-    """Task review states."""
+    """Task review states.
+
+    - ``PASSED`` - The task has been accepted in review.
+    - ``FAILED`` - The task has been rejected in review.
+    - ``CORRECTED`` - The task has been accepted with corrections made in the review.
```
Comment (ben-stewart-rb) [inline redbrick/common/enums.py:42]: The Task has been accepted with the corrections made in review.

File: redbrick/common/enums.py (hunk 3 above)
Comment (pritamrungta) [reply redbrick/common/enums.py:20]: nvm.. separated it into 2 lines

**Review by dereklukacs — APPROVED** [review]

**Pull request merged**